<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HFUT Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f0f0f0;
}

QWidget {
    font-family: &quot;SF Pro Display&quot;, &quot;Segoe UI&quot;, &quot;Arial&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #f0f0f0;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QFrame" name="displayFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: #000000;
    border-radius: 12px;
    border: 1px solid #333333;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="displayLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="modeInfoLayout">
         <item>
          <widget class="QLabel" name="modeLabel">
           <property name="text">
            <string>普通</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    color: #ffffff;
    font-size: 14px;
    font-weight: 500;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="baseLabel">
           <property name="text">
            <string>DEC</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    color: #888888;
    font-size: 12px;
    font-weight: 400;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="displayLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: transparent;
    border: none;
    color: #ffffff;
    font-size: 36px;
    font-weight: 300;
    padding: 0px;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="buttonFrame">
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QGridLayout" name="buttonGridLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnModeToggle">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>BIN</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #a6a6a6;
    color: #ffffff;
    border: none;
    border-radius: 8px;
    font-size: 18px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #b8b8b8;
}

QPushButton:pressed {
    background-color: #949494;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnClearEntry">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #a6a6a6;
    color: #ffffff;
    border: none;
    border-radius: 8px;
    font-size: 18px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #b8b8b8;
}

QPushButton:pressed {
    background-color: #949494;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnClear">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #a6a6a6;
    color: #ffffff;
    border: none;
    border-radius: 8px;
    font-size: 18px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #b8b8b8;
}

QPushButton:pressed {
    background-color: #949494;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnDivide">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>÷</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ff9500;
    color: #ffffff;
    border: none;
    border-radius: 8px;
    font-size: 24px;
    font-weight: 400;
}

QPushButton:hover {
    background-color: #ffad33;
}

QPushButton:pressed {
    background-color: #e6850e;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui> 